### PR TITLE
New Gearing

### PR DESCRIPTION
**Pull request by JoeJanickiChSh**
Increases speed for the new gearing on the intake. Also adds some flags to disable subsystems in the robot map.

### CODE REVIEW

**Comment by JoeJanickiChSh** [comment]
Whoops turns out I changed the wrong value for the speed. Will probably need more testing.

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 1]
```diff
@@ -14,9 +14,9 @@ public class Intake extends SmartSubsystemBase {
     private final SmartMotorController deploymentMotor;
     private final SmartMotorController rollerMotor;
 
-    private static final double ROLLER_SPEED = 0.75;
-    private static final double DEPLOY_EXTEND_SPEED = 0.3;
-    private static final double DEPLOY_RETRACT_SPEED = -0.3;
+    private static final double ROLLER_SPEED = 1.0;
+    private static final double DEPLOY_EXTEND_SPEED = 0.5;
```
Comment (bot190) [inline src/main/java/frc/robot/subsystems/Intake.java]: We can probably increase this even more.

**Review by bot190 — COMMENTED** [review]
Doh!

**Review by msoucy — APPROVED** [review]

**Review by bot190 — CHANGES_REQUESTED** [review]
Let's also lower the current limit significantly.

**Review by bot190 — APPROVED** [review]
We ended up using the 30A current limit at Governor's Cup. There's still a bunch of friction in the slides.

**Pull request merged**